iceoryx_posh/include/iceoryx_posh/internal/runtime/ipc_interface_base.hpp

```diff
@@ -29,10 +29,13 @@
 #include "iceoryx_hoofs/platform/time.hpp"
 #include "iceoryx_hoofs/platform/types.hpp"
 #include "iceoryx_hoofs/platform/unistd.hpp"
-#include "iceoryx_hoofs/posix_wrapper/named_pipe.hpp"
 #include "iceoryx_posh/iceoryx_posh_types.hpp"
 #include "iceoryx_posh/internal/runtime/ipc_message.hpp"
 
+#if defined(_WIN32)
+#include "iceoryx_dust/posix_wrapper/named_pipe.hpp"
+#endif
+
```

Comment:
And here we should include the `unix_domain_socket.hpp` for non-windows platforms